Formula/rio.rb

```diff
@@ -0,0 +1,25 @@
+class Rio < Formula
+  desc "Hardware-accelerated GPU terminal emulator powered by WebGPU"
+  homepage "https://raphamorim.io/rio/"
+  url "https://github.com/raphamorim/rio/archive/refs/tags/v0.0.7.tar.gz"
+  sha256 "5a3fcdc6bbad3274286bc41c5225c9ae576ffcd13b80649247656c0f41422de9"
+  license "MIT"
+  head "https://github.com/raphamorim/rio.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on :macos
```

Comment:
```suggestion
```
This doesn't seem correct, since your repository explicitly indicates Linux support.

Reply:
It does has Linux support but I removed it since there's 2 concerns:

- how would be possible to flag if is wayland or x11 https://raphamorim.io/rio/install/#building (otherwise will not work)
- prob can't test linux (same situation of darwin x86) due to containers not having GPU devices.

Reply:
Can we add a (brief) comment in the formula explaining why it has `depends_on :macos` then?

Reply:
For sure, updated 👌